src/lib/components/Cytoscape.react.js

```diff
@@ -676,7 +823,57 @@ Cytoscape.propTypes = {
          */
         boundingBox: PropTypes.object,
     }),
-
+    /**
+     * Define a custom context menu. The behaviour of each menu item can be defined in 1 of 3 ways.
```

Comment:
nitpickyyyy text edits:
- hyphenate `built-in`
- capitalize `Javascript`
- capitalize `Dash`
- put a space at the end of each line of this comment; that way if the line breaks get removed (which happens for some of the docstrings) the formatting will still look OK.